core-web/libs/sdk/angular/next/components/dotcms-editable-text/dotcms-editable-text.component.ts

```diff
@@ -45,7 +40,9 @@ import { TINYMCE_CONFIG, DOT_EDITABLE_TEXT_FORMAT, DOT_EDITABLE_TEXT_MODE } from
         {
             provide: TINYMCE_SCRIPT_SRC,
             useFactory: () => {
-                return `${DotCmsClient.dotcmsUrl}/ext/tinymcev7/tinymce.min.js`;
+                const { dotCMSHost } = getUVEState() || {};
+
+                return `${dotCMSHost}/ext/tinymcev7/tinymce.min.js`;
```

Comment:
should we add a console.warn or error that says that it does not have any host and therefore the component will not render as an editable text?

Reply:
Sure!

Reply:
Good catch!
I added the logic in the component to avoid complicating the useFactory!

https://github.com/dotCMS/core/blob/6bfcead70db9bf16a9857bb586aee58c2425e7c4/core-web/libs/sdk/angular/next/components/dotcms-editable-text/dotcms-editable-text.component.ts#L159-L177